Nemo Connect/Sources/NCNetworkSessionManager.m

```diff
@@ -207,4 +212,12 @@ - (void)URLSession:(NSURLSession *)session dataTask:(NSURLSessionDataTask *)data
     [delegate URLSession:session dataTask:dataTask didReceiveData:data];
 }
 
+- (void)URLSessionDidFinishEventsForBackgroundURLSession:(NSURLSession *)session
+{
+    if (self.backgroundTaskFinishedCompletionBlock)
+    {
+        self.backgroundTaskFinishedCompletionBlock();
```

Comment:
not sure how it is working, but is it necessary to pass back the NSURLSession object? (for example to get URL of the downloaded data)